dockstore-yaml-proposals/jbrowse/healthcheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+set -e
+curl -s -o /dev/null --fail localhost:${HTTP_PORT:-3000}/data/tracks.conf
```

Comment:
Having some schematization here will be important in order to be able to use the response.

Reply:
For now, going with simple exit codes. Just like ports, I would prefer to tackle more complex cases when we run across them.